tests/libs/util/wer_report.hpp

```diff
@@ -84,8 +86,14 @@ class _wer_report
     static constexpr const wchar_t wer_event_type[] = L"Test Application Crash";
 
     static int __CRTDECL
-    _terminate_hook(int, char*, int*)
+    _terminate_hook(int reportType, char* message, int* returnValue)
```

Comment:
`_terminate_hook` is called before the default debug handling of the CRT kicks in. So writing to stderr here + aborting the process is the same as setting the report modes.